exercises/G-numbers/exercise.js

```diff
@@ -1 +1,12 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var studentsAndMentors = numberOfStudents + numberOfMentors;
+
+var messageStudents = "Number of students: " + numberOfStudents;
```

Comment:
This is great javascript basics. You understanding the basic concept to declaring variables and assigning arguments to them. Well done!